packages/sources/cryptoapis-v2/src/config.ts

```diff
@@ -2,11 +2,35 @@ import { Requester } from '@chainlink/ea-bootstrap'
 import { Config } from '@chainlink/types'
 
 export const DEFAULT_ENDPOINT = 'price'
-export const NAME = "CRYPTOAPIS_V2"
+export const NAME = 'CRYPTOAPIS_V2'
 
 export const makeConfig = (prefix?: string): Config => {
   const config = Requester.getDefaultConfig(prefix, true)
   config.api.headers['X-API-Key'] = config.apiKey
   config.api.baseURL = 'https://rest.cryptoapis.io'
   return config
 }
+
+export const BLOCKCHAIN_NAME_BY_TICKER = {
+  btc: 'bitcoin',
+  eth: 'ethereum',
+  ltc: 'litecoin',
+  etc: 'ethereum-classic',
+  bch: 'bitcoin-cash',
+  dash: 'dash',
+  doge: 'dogecoin',
+} as const
+
+export function isCoinType(key: string): key is keyof typeof BLOCKCHAIN_NAME_BY_TICKER {
+  return !!BLOCKCHAIN_NAME_BY_TICKER[key.toLowerCase()]
```

Comment:
```suggestion
  return !!BLOCKCHAIN_NAME_BY_TICKER[key.toLowerCase() as keyof typeof BLOCKCHAIN_NAME_BY_TICKER]
```

Feel free to make a type alias like so, if you find the verbosity too high
```ts
type BlockchainTickers = keyof typeof BLOCKCHAIN_NAME_BY_TICKER
```